models/llama3_3/prompt_format.md

```diff
@@ -72,14 +72,135 @@ Here's my response
 
 ## Tool Calling Formats
 
+Here we describe how to invoke the Llama 3.3 instruction tuned model for tool-calling (also called function-calling). We recommend zero-short function calling over built-in tools.
+
+### Zero shot function calling
+
+
+For Llama3.3 70B instruct models, we are continuing the format for zero shot function calling, introduced in llama 3.2.
+This format is designed to be more flexible and powerful than the format in 3.1.
+All available functions can be provided in the system message.
+
+Here is an example for the same,
+
+
+##### Input Prompt Format
+```
+<|begin_of_text|><|start_header_id|>system<|end_header_id|>
+
+You are an expert in composing functions. You are given a question and a set of possible functions.
+Based on the question, you will need to make one or more function/tool calls to achieve the purpose.
+If none of the function can be used, point it out. If the given question lacks the parameters required by the function,
```

Comment:
Change "none of the function can" to "none of the function(s) can" (as we do further down in the prompt).

I am thinking that this text is within the prompt, so we might not want to perturb it.

Reply:
Since its within the prompt-we should not change it IMO

Reply:
Yeah, lets not change this.